tests/fixtures/require-spread/bad/example.js

```diff
@@ -1,3 +1,5 @@
 const wrap = (f, g) => {
-  return (...args) => g.apply(g, [f].concat(args));
-}
+  return (...args) => g.apply(null, [f].concat(args));
+};
+
+wrap(1, (a) => a + 2);
```

Comment:
Does this specific line play a role in the test failing or passing? Or is it just there to illustrate that `g` is a function?

Reply:
If we don't add this line, then eslint would fail because wrap is an unused variable, not necessarily because of the spread.